pages/vi/vi-faq.md

```diff
@@ -83,13 +83,13 @@
 
 #### Q14: When I first run BeLL with the "vagrant up" command, why does the download fail?
 
-+ You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar:
++ You should first check if [Hashicorp's atlas](https://status.hashicorp.com) is up and running by looking at the status bar just below the logo:
```

Comment:
It's still saying "Hashicorp's atlas" here. To be precise, we should be looking at "Vagrant Cloud" status that handles "Vagrant box publishing, hosting, and management", right?

A closer screenshot only showing "Vagrant Cloud" status would also help.